frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelinesTopology.cy.ts

```diff
@@ -466,6 +466,69 @@ describe('Pipeline topology', () => {
         .findValue()
         .contains('False');
     });
+
+    it('Test pipeline triggered run bottom drawer output', () => {
+      initIntercepts();
+      pipelineRunDetails.visit(projectId, mockRun.run_id);
+
+      pipelineRunJobDetails.findBottomDrawer().findBottomDrawerYamlTab().click();
+      pipelineRunDetails.findBottomDrawer();
+      pipelineDetails.findYamlOutput().click();
+      const pipelineDashboardCodeEditor = pipelineDetails.getPipelineDashboardCodeEditor();
+      pipelineDashboardCodeEditor.findInput().should('not.be.empty');
+    });
+  });
+
+  describe('Pipeline run Input/Output', () => {
+    beforeEach(() => {
+      initIntercepts();
+      pipelineRunDetails.visit(projectId, mockRun.run_id);
+    });
+
+    it('Test with input/output artifacts', () => {
+      pipelineRunDetails.findTaskNode('create-dataset').click();
+      pipelineRunDetails.findRightDrawer().findRightDrawerInputOutputTab().should('be.visible');
```

Comment:
store this in a variable
```suggestion
      const rightDrawer=pipelineRunDetails.findRightDrawer();
      rightDrawer.findRightDrawerInputOutputTab().should('be.visible');
```